cdc/entry/mounter.go

```diff
@@ -166,7 +172,23 @@ func (m *mounterImpl) unmarshalAndMountRowChanged(ctx context.Context, raw *mode
 			if rowKV == nil {
 				return nil, nil
 			}
-			return m.mountRowKVEntry(tableInfo, rowKV, raw.ApproximateDataSize())
+			row, err := m.mountRowKVEntry(tableInfo, rowKV, raw.ApproximateDataSize())
+			if err != nil {
+				return nil, err
+			}
+			// Now we need to filter a row here is because we need its tableInfo
+			ignore, err := m.filter.ShouldIgnoreDMLEvent(row, tableInfo.TableInfo)
```

Comment:
should you add some statistical informations about the ignored events?

Reply:
Good advice!